app/uk/gov/hmrc/gatekeeperemail/repositories/DraftEmailRepository.scala

```diff
@@ -67,11 +68,11 @@ class DraftEmailRepository @Inject() (mongoComponent: MongoComponent, appConfig:
         fromRegistries(
           fromCodecs(
             Codecs.playFormatCodec(domainFormat),
-            Codecs.playFormatCodec(EmailMongoFormatter.emailTemplateDataFormatter),
-            Codecs.playFormatCodec(EmailMongoFormatter.cargoFormat),
-            Codecs.playFormatCodec(EmailMongoFormatter.attachmentDetailsFormat),
-            Codecs.playFormatCodec(EmailMongoFormatter.attachmentDetailsWithObjectStoreFormat),
-            Codecs.playFormatCodec(EmailMongoFormatter.emailFormatter)
+            Codecs.playFormatCodec(EmailTemplateData.format),
```

Comment:
I wonder if the original devs went a bit cautious in stuffing in every "codec".  I don't think it's needed unless you're actually trying to use them IIRC.  Might be worth checking locally.  Could be case of copy paste noodling bad habit.

Reply:
Most of these relate to adding attachments to emails which we don't want to touch because it's an unknown area, which I think might never have worked.